file/fat32.lisp

```diff
@@ -539,11 +538,10 @@ Valid media-type ara 'FAT32   ' " fat-type-label)))
     (do ((cluster-n start-cluster (fat-value fat cluster-n))
          (n-cluster 0 (1+ n-cluster)))
         ((>= cluster-n (last-cluster-value ffs)) result)
-      (block-device-read disk
-                         (first-sector-of-cluster ffs cluster-n)
-                         spc
-                         result
-                         :offset (* n-cluster spc sector-size)))))
+      (replace result (block-device-read-sector disk
+                                                (first-sector-of-cluster ffs cluster-n)
+                                                spc)
+               :start1 (* n-cluster spc sector-size)))))
```

Comment:
I don't understand this change. The original code could read directly into the result array.

Reply:
It uses page cache, it to much work to make fat32 work properly with clusters.